shell/client/styles/_topbar.scss

```diff
@@ -75,8 +75,7 @@ body>.topbar {
   background-color: $topbar-background-color;
   color: #ccc;
 
-  // Using 0 rather than CSS default ("auto") allows introjs to highlight elements in the topbar.
-  z-index: 0;
+  z-index: 1;
```

Comment:
That sounds like the right choice for now, then.

Someday we can figure out either transcluding `modalDialogWithBackdrop` or making topbar `popupTemplate` have a way to do a better job of isolating the popup from whatever is in the background.  For instance, today the popups block scroll events in the grain-frame, but don't give any visual indication that that should be the case.  And we've wanted the login-nag to be auto-displayed, but to not be application-modal either...

I suspect we'll eventually want a unified way to support at least two behaviors - modal dialog with backdrop (for all the prompt/confirm/share/etc transactional interactions), and toggleable dropdown (for notifications and the account/login menu).